services/identity/service_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 func TestMain(m *testing.M) {
 	log.Info("Skipping because of bftcosi and skipchain - #482")
-	log.MainTest(m)
+	//log.MainTest(m)
```

Comment:
Is this on purpose? Can we have a comment with reference to some issue (or a skip message?)